src/Powercord/plugins/pc-routelinks/index.js

```diff
@@ -0,0 +1,83 @@
+const { Plugin } = require('powercord/entities');
+const { getRepoInfo, cloneRepo } = require('../pc-moduleManager/util');
+const express = require('express');
+const { open: openModal, close: closeModal } = require('powercord/modal');
+const { React, getModule } = require('powercord/webpack');
+const Modal = require('./components/ConfirmModal');
+
+
+module.exports = class RDLinks extends Plugin {
+  async startPlugin () {
+    this.queue = new Set();
+    this.app = express();
+    this.showNotification = await getModule([ 'showNotification' ]);
+    this.app.get('/', (req, res) => {
+      res.send('Rise and shine, Mister Freeman. Rise and... shine. Not that I... wish to imply you have been sleeping on the job. No one is more deserving of a rest... and all the effort in the world would have gone to waste until... well, let\'s just say your hour has... come again. The right man in the wrong place can make all the difference in the world. So, wake up, Mister Freeman. Wake up and... smell the ashes...');
+    });
+
+    this.app.get('/install/', async (req, res) => {
+      this.info = await getRepoInfo(req.query.address);
+      if (this.info) {
+        if (this.info.isInstalled) {
+          res.send(`${this.info.type} ${this.info.repoName} is already installed!`);
+          return;
+        }
+        res.send(`Sent prompt to install ${req.query.address}`);
+        this.info.url = req.query.address;
+        this.openInstallModal();
+      } else {
+        res.send(`Cannot find repository: ${req.query.address}`);
+      }
+    });
+
+
+    this.httpserv = this.app.listen(6473);
```

Comment:
what if this port is already taken? What if you run two instances of powercord at once?

Reply:
Yeah that's what i was thinking, having a port range defeats the purpose

Reply:
There would kind of be the same problem with protocols as well, since only one instance can control it

Reply:
at least we know the deeplink wouldn't be taken